lib/mix/tasks/install.ex

```diff
@@ -334,12 +342,30 @@ defmodule Mix.Tasks.Beacon.Install do
   end
 
   defp validate_options!(options) do
-    site = String.to_atom(options[:site])
+    site = options |> Keyword.get(:site, "false") |> String.to_atom()
+    path = options |> Keyword.get(:path, "/#{site}")
```

Comment:
Actually you can just use the value from `options` directly and let `Beacon.Types.Site` do the actual validation.

```suggestion
```

And then:

```elixir
!Beacon.Types.Site.valid?(options[:site]) ...
```

Reply:
https://github.com/BeaconCMS/beacon/pull/387/commits/162a51ad319d3ed83e7741be375e188e849738ed